esphome/es32-c3-mini--goe22-controll.yaml

```
@@ -127,20 +127,20 @@ button:
 
 switch:
   - platform: template
-    optimistic: True
+    optimistic: true
     restore_mode: RESTORE_DEFAULT_OFF
     id: low_amp_flag
     name: Low Amps Flag ${variant}
     internal: False
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)